mobile/src/stateMachine/views/strategies/ChooseStrategiesView.tsx

```diff
@@ -0,0 +1,217 @@
+import { observable } from 'mobx';
+import { observer } from 'mobx-react';
+import React from 'react';
+import { Dimensions, FlatList, StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import { Button, Container, MasloPage, StrategyCard } from 'src/components';
+import AppController from 'src/controllers';
+import TextStyles from 'src/styles/TextStyles';
+import AppViewModel from 'src/viewModels';
+import Colors from '../../../constants/colors/Colors';
+import { ScenarioTriggers } from '../../abstractions';
+import { ViewState } from '../base';
+
+const minContentHeight = 300;
+const { width } = Dimensions.get('window');
+
+@observer
+export class ChooseStrategiesView extends ViewState {
+
+  @observable
+  private dropDownIsExtended = false;
+  private domains: string[] = [];
+  private currentFilter: string = null;
+
+    constructor(props) {
+        super(props);
+        this._contentHeight = this.persona.setupContainerHeightForceScrollDown({ transition: { duration: 0} });
+        this.hidePersona();
+        this.domains = this.viewModel.selectedDomains;
+
+        this.onLearnMorePress = this.onLearnMorePress.bind(this);
+        this.onSelectStrategy = this.onSelectStrategy.bind(this);
+    }
+
+    public get viewModel() {
+        return AppViewModel.Instance.ChooseStrategy;
+    }
+
+    async start() {
+      //
```

Comment:
...

Reply:
start has to be implemented otherwise ` Non-abstract class 'ChooseStrategiesView' does not implement inherited abstract member 'start' from class 'ViewState<{}, any>' `